### PR TITLE
make-default-user-account-using-faker-and-seeder

### PR DESCRIPTION
**Pull request by axnish27**
db seeded using faker

### CODE REVIEW

File: database/seeders/DatabaseSeeder.php
Comment (Raza9798) [inline database/seeders/DatabaseSeeder.php:1]: Does this factory method creates a user with requested user information #5 @axnish27

**Review by Raza9798 — CHANGES_REQUESTED** [review]

File: database/seeders/DatabaseSeeder.php
Comment (axnish27) [reply database/seeders/DatabaseSeeder.php:1]: i implemented it now , please check again

File: database/seeders/DatabaseSeeder.php [hunk 1]
```diff
@@ -16,8 +16,9 @@ public function run(): void
         // User::factory(10)->create();
 
         User::factory()->create([
-            'name' => 'Test User',
-            'email' => 'test@example.com',
+            'name' => 'admin',
+            'email' => 'admin@example.com',
+            'password' => 'password'
```
Comment (Raza9798) [inline database/seeders/DatabaseSeeder.php:21]: plain text passwords can be `bruteforce` easily. please encrypt the password using `bcrypt` mechanism

**Review by Raza9798 — CHANGES_REQUESTED** [review]

File: database/seeders/DatabaseSeeder.php (hunk 1 above)
Comment (axnish27) [reply database/seeders/DatabaseSeeder.php:21]: okay sure got it, will use bcrypt #7

**Review by Raza9798 — APPROVED** [review]

**Pull request merged**